MH4U Dex/Images.xcassets/Region Icons/ancestral_steppe-icon.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x",
+      "filename" : "Image.png"
```

Comment:
Thanks for this catch!  It came from a wiki with CC-BY-SA 3.0.  It's fine to include it, but I'll have to include an attributions page somewhere.

I think I should go through every image I have, and every Image I plan to add, and check the license on all of them.